.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,86 @@
+name: Continuous integration
+
+on:
+  workflow_dispatch:
```

Comment:
Would the workflow_dispatch be available to anyone on the internet since this is open source? It would probably be a bad idea to allow anyone in the world to run a job on bob.

Reply:
We can remove it, however I think that running a github action requires some kind of permission on the repository.